functions/api/image/tokens/tokenImage.test.ts

```diff
@@ -19,5 +19,5 @@ const invalidTokenImageUrl = [
 
 test.each(invalidTokenImageUrl)('invalidAssetImageUrl', async (url) => {
   const response = await fetch(new Request(url))
-  expect(response.status).toBe(404)
+  expect([404, 500]).toContain(response.status)
```

Comment:
one of these is a flake, and one of these is the actual expectation.
this will make it less flaky, but at the expense of not actually running the test.

could you retry if it is a 500?